tox.ini

```diff
@@ -25,20 +25,23 @@ commands =
     check-manifest
     python setup.py check -m -r -s
 
-[testenv:pep8]
+[testenv:lint-py2]
 basepython = python2
-deps = flake8==3.3.0
-commands = flake8 .
-
-[testenv:py3pep8]
-basepython = python3
-deps = flake8==3.3.0
-commands = flake8 .
+deps =
+    flake8==3.3.0
+    isort==4.2.5
+commands =
+    flake8 .
+    isort --recursive --check-only --diff pip tests
 
-[testenv:isort]
+[testenv:lint-py3]
 basepython = python3
-deps = isort==4.2.5
-commands = isort --recursive --check-only --diff pip tests
+deps =
+    flake8==3.3.0
+    isort==4.2.5
+commands =
+    flake8 .
+    isort --recursive --check-only --diff pip tests
```

Comment:
Running isort twice, once on Py3 and once on Py2 can't hurt. I'll remove this if someone feels otherwise strongly.